pkg/notifications/shoutrrr.go

```diff
@@ -109,10 +110,22 @@ func getShoutrrrTemplate(c *cobra.Command) *template.Template {
 
 	tplString, err := flags.GetString("notification-template")
 
+	funcs := template.FuncMap{
+		"ToUpper": func(input interface{}) string {
+			return strings.ToUpper(fmt.Sprintf("%v", input))
+		},
+		"ToLower":func(input interface{}) string {
+			return strings.ToLower(fmt.Sprintf("%v", input))
+		},
+		"Title":func(input interface{}) string {
+			return strings.Title(fmt.Sprintf("%v", input))
+		},
```

Comment:
It can be done much simpler.
```suggestion
		"Title": strings.Title,
```
This was actually the [example from the documentation](https://golang.org/pkg/text/template/#example_Template_func) 😁